apps/admin/app/layout.tsx

```diff
@@ -13,7 +13,7 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body>
+      <body className="h-screen w-full">
```

Comment:
import {Body} from "@repo/ui/xyz"
```suggestion
      <Body className="h-screen w-full">
```